app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -186,6 +186,32 @@ describe("FormAudience", () => {
     ).toHaveAttribute("disabled");
   });
 
+  it("enables language field for mobile", async () => {
+    render(
+      <Subject
+        experiment={{
+          ...MOCK_EXPERIMENT,
+          application: NimbusExperimentApplicationEnum.FENIX,
+        }}
+      />,
+    );
+
+    expect(screen.queryByTestId("languages")).toBeInTheDocument();
+  });
+
+  it("disables language field for desktop", async () => {
+    render(
+      <Subject
+        experiment={{
+          ...MOCK_EXPERIMENT,
+          application: NimbusExperimentApplicationEnum.DESKTOP,
+        }}
+      />,
+    );
+
+    expect(screen.queryByTestId("languages")).not.toBeInTheDocument();
+  });
```

Comment:
Might also be worth asserting the opposite in each test, ie languages is shown, locales isn't, and visa versa?

Reply:
absolutely, that check would be better to have, I just added it :)